docs/embedding/sdk/quickstart-cli.md

```diff
@@ -25,65 +27,86 @@ Change into your React application and run:
 npx @metabase/embedding-sdk-react@latest start
 ```
 
-The tool will walk you through the setup. There are a fair number of pieces to put together, so here's an overview of what the command does.
+This CLI tool will walk you through the setup. There are a fair number of pieces to put together, so here's an overview of what the command does:
+
+- [Prereq check](#prereq-check)
+- [Database connection (optional)](#database-connection-optional)
+- [Metabase setup](#metabase-setup)
+- [Permissions setup with multi-tenancy (optional)](#permissions-setup-with-multi-tenancy-optional)
+- [React components setup](#react-components-setup)
+- [Behold: Metabase is embedded in your app](#start-your-app)
+
+## Prereq check
+
+The tool will check for the following:
+
+- You've run the command in the top-level directory of your React application.
+- You've installed the SDK (if you haven't, the CLI will install the SDK for you and add it as a dependency in your `package.json`).
+- You have Docker up and running on your machine.
+
+## Database connection (optional)
 
-## Checks for prereqs
+The tool will ask if you have a database to connect to. Use the arrow keys to select Yes or No. The tool will use this database to generate an embedded dashboard.
```

Comment:
The sentence `The tool will use this database to generate an embedded dashboard.` comes right after "select yes or no", so to me it reads that this is an immediate consequence of selecting yes or no, instead of something that happens at the end. I think even just swapping the two sentences could work.

Reply:
Or just

`The tool will ask if you have a database to connect to. If you don't have one it will use Metabase's sample DB instead.`